Integrate oegenerate to oeutil (#4067)
openenclave/openenclave: CHANGELOG.md

- Add the following attestation claims from oe_verify_evidence():
  - OE_CLAIM_TCB_STATUS
  - OE_CLAIM_TCB_DATE
- Publish tool `oeutil`.

>   - The tool, currently under the tools directory, is to [integrate multiple OE utilities](tools/oeutil/README.md) in the future.

Reviewer: ```suggestion
  - The tool, currently under the tools directory, will [integrate multiple OE utilities](tools/oeutil/README.md) in the future.
```
